Decred Wallet/Features/More/MoreMenuItem.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  MoreMenuItem.swift
+//  Decred Wallet
+//
+//  Created by Suleiman Abubakar on 03/12/2019.
+//  Copyright © 2019 Decred. All rights reserved.
+//
+
+import UIKit
+
+enum MoreMenuItem: String, CaseIterable {
+    case settings = "Settings"
+    case security = "Security"
```

Comment:
rawvalue should be `Security Tools`

Reply:
Should just rename the entire enum case:
```suggestion
    case securityTools = "Security Tools"
```